client-spark/shuffle-manager-2/src/main/java/org/apache/spark/shuffle/rss/HashBasedShuffleWriter.java

```diff
@@ -17,17 +17,14 @@
 
 package org.apache.spark.shuffle.rss;
 
+import com.aliyun.emr.rss.client.ShuffleClient;
+import com.aliyun.emr.rss.client.write.DataPusher;
+import com.aliyun.emr.rss.common.RssConf;
```

Comment:
tried to keep import orders, but failed, guess it does not compatible w/ google style

Reply:
ref: https://github.com/diffplug/spotless/tree/main/plugin-maven#java

Reply:
I figured it out, I should define the `<importOrder>` part to the last step